torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9327,6 +9348,57 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                # This method not supported by TorchScript
                DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
            )),
+    OpInfo('empty_like',
+           dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
+           supports_out=False,
+           sample_inputs_func=sample_inputs_like_fns,
+           supports_autograd=False,
+           skips=(
+               # This method not supported by TorchScript
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
```

Comment:
See https://github.com/pytorch/pytorch/pull/64282#discussion_r722917935.

```suggestion
               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit', dtypes=(torch.float32, torch.complex64)),
```